assets/prefabs/structures/Barn.prefab

```diff
@@ -0,0 +1,97 @@
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.BACK", "region": { "min": [-2, 9, 29], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.BACK", "region": { "min": [-1, 10, 28], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.BACK", "region": { "min": [0, 11, 27], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.RIGHT", "region": { "min": [4, 9, 29], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.RIGHT", "region": { "min": [3, 10, 28], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.RIGHT", "region": { "min": [2, 11, 27], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.RIGHT", "region": { "min": [5, 8, 30], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slopecorner.BACK", "region": { "min": [-3, 8, 30], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slope.BACK", "region": { "min": [-1, 9, 29], "size": [5, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slope.BACK", "region": { "min": [-2, 8, 30], "size": [7, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slope.BACK", "region": { "min": [0, 10, 28], "size": [3, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof:engine:slope.BACK", "region": { "min": [1, 11, 27], "size": [1, 1, 1]}},
+            { "blockType": "HjerlHede:algaeRoof", "region": { "min": [1, 11, 4], "size": [1, 1, 22]}},
+            { "blockType": "engine:air", "region": { "min": [3, 1, 0], "size": [4, 4, 1]}},
+            { "blockType": "engine:air", "region": { "min": [7, 1, 14], "size": [1, 4, 3]}},
+            { "blockType": "CoreBlocks:DoorBottom.BACK", "region": { "min": [3, 1, 0], "size": [2, 2, 1]}},
+            { "blockType": "CoreBlocks:DoorTop.BACK", "region": { "min": [3, 3, 0], "size": [2, 2, 1]}},
+            { "blockType": "HjerlHede:grill.LEFT", "region": { "min": [2, 2, 1], "size": [1, 5, 28]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 7, 1], "size": [1, 1, 28]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 1, 1], "size": [1, 1, 28]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 2, 4], "size": [1, 5, 1]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 2, 9], "size": [1, 5, 1]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 2, 14], "size": [1, 5, 1]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 2, 19], "size": [1, 5, 1]}},
+            { "blockType": "CoreBlocks:Plank", "region": { "min": [2, 2, 24], "size": [1, 5, 1]}},
+            { "blockType": "HjerlHede:diagonalRight", "region": { "min": [-4, 2, 0], "size": [1, 1, 1]}},
```

Comment:
It might work to just search and replace `HjerlHede:diagonalRight` with `StructuralResources:TimberFrameDiagonalRight` - maybe you can give it a try? 🙃